src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj

```diff
@@ -1,17 +1,26 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <!-- NuGet -->
     <Version>1.25.2-prerelease</Version>
     <Title>Datadog APM - OpenTracing</Title>
     <Description>Provides OpenTracing support for Datadog APM</Description>
     <PackageTags>$(PackageTags);OpenTracing</PackageTags>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
We want to add `ContinuousIntegrationBuild` in gitlab builds, too. Add as property here? https://github.com/DataDog/dd-trace-dotnet/blob/eed37a0999a38804105b5c4ed2a7b0f1fcd5bd54/tasks/dotnettracer.py#L41

Reply:
Unless there's a way to detect gitlab builds like you did with `TF_BUILD`.

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1401/commits/1e7fcf739a4acd23493549956a6c6aba69ac5b37
Is there any way to test it's properly working? 🤔

Reply:
We can trigger a gitlab build and check the output.

Reply:
To minimize changes in the GitLab steps, we should check `'$(GITLAB_CI)' == 'true'` in this csproj (source: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html)

Reply:
What do you think of moving the PropertyGroup that sets the ContinuousIntegrationBuild variable into the parent `Directory.Build.props` so all of the projects receive the same treatment?